docker.local/config/sc.yaml

```diff
@@ -247,11 +247,13 @@ smart_contracts:
       delete: 100
       vestingsc-update-settings: 100
   zcnsc:
+    owner_id: 1746b06bb09f55ee01b33b5e2e055d6cc7a900cb57c0a3a5eaabb8a0e7745802
     min_mint_amount: 1
-    percent_authorizers: 0.7
-    min_authorizers: 1
     min_burn_amount: 1
     min_stake_amount: 0
+    min_lock_amount: 0
```

Comment:
Use `min_lock` and `min_stake` to keep the consistent naming as other SC settings.

Reply:
done